builder/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<artifactId>parent</artifactId>
 		<groupId>cz.xtf</groupId>
-		<version>0.20-SNAPSHOT</version>
+		<version>0.21-SNAPSHOT</version>
```

Comment:
Hi @mnovak1 I don't know the process so I updated the version so that just who want to swich to this new feature can, but I can easily revert to 0.20-SNAPSHOT